community/README.md

```diff
@@ -3,3 +3,8 @@
 Welcome to the Argo community!
 
 Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done. [This document](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure of the Argo Community including the roles, responsibilities and processes that govern Argo projects and community.
+
+## Argo Workflows Sustainability Effort
```

Comment:
We can also link this in contributing guide

Reply:
good idea, will do